### ISSUE TITLE
Investigate the need (or not) of EULA agreement of contributors

### ISSUE THREAD

**Issue by agardelakos** [panoptes-organization/panoptes#63]
Maybe we need contributors to sign some kind of EULA agreement before we start reviewing their PR (or even at the creation of a ticket). Real life example of the agreement is [this](https://cla-assistant.io/KasperskyLab/Kaspresso?pullRequest=38) and an example use case is [this](https://github.com/KasperskyLab/Kaspresso/pull/52).

This ticket is not about writing code, only to track and document the investigation of the above

**Comment by fgypas** [comment]
I am closing this for now, as it's not a priority.
